webplugin/js/app/zendesk-chat-service.js

```diff
@@ -28,10 +28,26 @@ function ZendeskChatService() {
             return;
         }
         window.console.log("handleUserMessage: ", event);
-        zChat.sendChatMsg(event.message.message, function (err, data) {
-            window.console.log("zChat.sendChatMsg ", err, data)
-        });
-        // TODO look for attachment
+        if(event.message.contentType == KommunicateConstants.MESSAGE_CONTENT_TYPE.DEFAULT){
+            zChat.sendChatMsg(event.message.message, function (err, data) {
+                window.console.log("zChat.sendChatMsg ", err, data)
+            });
+
+        }else if(event.message.contentType == KommunicateConstants.MESSAGE_CONTENT_TYPE.ATTACHMENT){
+
+            let fileInputElement=document.getElementById("mck-file-input");
```

Comment:
i guess let wouldn't work in chat widget code, @Prochnost55 @gargshantnu can you guys confirm?

Reply:
'let' is working, when running SDK locally.

Reply:
in IE? can you check it in opera mini as well
https://caniuse.com/?search=let

Reply:
Tested it in Opera, its working fine there 
In the below screenshot you can see in logs that "File has been sent successfully"
<img width="960" alt="image" src="https://user-images.githubusercontent.com/36797892/147818658-3a7d82d3-5f70-45c1-aa3e-a7bafdd9284b.png">

And as far IE is concerned, in my OS (Windows 11) IE is not supported. So have to ask Vipin to do that while he tests.

Reply:
why do we need to wait for Vipin on this? check https://www.lambdatest.com/ i think we have a subscription for it as well, check in QA channel

Reply:
Yes it was giving an error for let keyword. So changed it to "var"